inference/models/starcoder.yaml

```diff
@@ -0,0 +1,22 @@
+port: 8000
+name: starcoder
+route_prefix: /starcoder
+cpus_per_worker: 24
+gpus_per_worker: 0
+deepspeed: false
+workers_per_group: 2
+ipex:
+  enabled: false
+  precision: bf16
+device: "cpu"
+model_description:  
+  model_id_or_path: bigcode/starcoder
+  tokenizer_name_or_path: bigcode/starcoder
+  chat_processor: ChatModelGptJ
+  prompt:
+    intro: ''
+    human_id: ''
+    bot_id: ''
+    stop_words: []
+  config:
+    use_auth_token: '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'
```

Comment:
`use_auth_token` cannot be written directly in config yaml, it needs to be set in CI file. @jiafuzha please help confirm this.

Reply:
yes, strictly speaking. we cannot. But it's only read-only key. If it can pass github security check, I think we can leave it for now.

Reply:
If the environment of our CI nodes have `env.HF_ACCESS_TOKEN` configured. I think I can try to get `use_auth_token` runtime from env instead of passing in yaml directly.

Reply:
yes, it's better. 

Maybe we can have a unit test later to verify if use_auth_token is passed correctly. This ticket exposed and fixed the token bug in several places, which shows the right value of CI.

Reply:
Ok, removed them from CI. Let's merge this first and I will create a follow-up PR to hide auth token.